_i18n/es.yml

```diff
@@ -341,3 +341,10 @@ latest-publications:
   posts: Últimos posts
 author:
   publications-title: Publicaciones
+thanks:
+  get-back-soon: Agradecemos que se haya tomado el tiempo de escribirnos. Nos pondremos en contacto contigo pronto.
+  our-blog: Nuestros posts
+  video-section: Sección de videos
+  services-we-offer: Servicios que ofrecemos
+  until-then: >
+    Hasta entonces, puedes consultar algunos de nuestros recursos:
```

Comment:
"Nos pondremos en contacto lo antes posible" instead of "nos pondremos en contacto contigo pronto".

Sounds better I'd say